app/main.py

```diff
@@ -1,2 +1,78 @@
+from typing import Any, Hashable
+
+
+class Node:
+    def __init__(
+        self,
+        key: Hashable,
+        value: Any
+    ) -> None:
+        self.key = key
+        self.value = value
+        self.next = None
```

Comment:
You should store hash(key) value in Node, as it works in dictionaries in Python
Also u can remove next from here, store just key, hash and value

Reply:
I can't remove 'next' because of AttributeError: 'Node' object has no attribute 'next'